task: update documentation

o11yguru opened the pull request:
Documentation updates for March release

o11yguru committed:
task: update documentation

o11yguru committed:
task: remove old src folder

o11yguru committed:
task: update doucmentation with ingresss install guide

o11yguru committed:
task: update doucmentation aws install steps

o11ika commented on src/install/aws.md:
```diff
@@ -0,0 +1,141 @@
+
+## Setup the MDAI Engine© in AWS
+
+You are going to learn to do the following in less than five minutes:
```
I don't think it's doable in less than 5 minutes

o11ies commented on src/install/aws.md:
```diff
@@ -0,0 +1,141 @@
+		CDK_NEW_BOOTSTRAP=1 $(CDK) bootstrap
+
+# Bootstrap your environment
+cdk bootstrap aws://<your_aws_account>/us-east-1 --profile <your_aws_profile>
```
should it also be <your_region>?

o11ika replied:
```suggestion
cdk bootstrap aws://<your_aws_account>/<your_region> --profile <your_aws_profile>
```

o11ika commented on src/install/aws.md:
````diff
@@ -0,0 +1,141 @@
+
+* Install [Go](https://go.dev/dl/) (1.20 or higher) from source or use homebrew  `brew install go`
+* [GOBIN environment variable](https://pkg.go.dev/cmd/go#hdr-Environment_variables) is set; if unset, initialize it appropriately, for example:
+```
````
```suggestion
```shell
```

o11ika commented on src/install/aws.md:
````diff
@@ -0,0 +1,141 @@
+
+* Install [AWS SSO](https://docs.aws.amazon.com/cli/latest/userguide/sso-configure-profile-token.html)
+* Login via the CLI
+```@bash
````
```suggestion
```shell
```

o11ika commented on src/install/aws.md:
````diff
@@ -0,0 +1,141 @@
+
+### AWS CDK
+Follow the steps in the [AWS CDK Install Guide](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install). Don't forget to [Bootstrap](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_bootstrap) your environment!
+```@bash
````
```suggestion
```shell
```

o11ika commented on src/install/aws.md:
````diff
@@ -0,0 +1,141 @@
+
+### Configure the MDAI© Engine
+
+```@bash
````
```suggestion
```shell
```

o11ika commented on src/install/aws.md:
````diff
@@ -0,0 +1,141 @@
+
+### Deploy the MDAI© Engine
+
+```@bash
````
```suggestion
```shell
```

o11ika commented on src/install/aws.md:
````diff
@@ -0,0 +1,141 @@
+
+
+Ensure your cluster is up and running.
+```@bash
````
```suggestion
```shell
```

o11ies commented on src/install/aws.md:
```diff
@@ -0,0 +1,141 @@
+
+Make sure that your developer environment has the following. This page assumes that you’re using bash. Adapt configuration and commands as necessary for your preferred shell.
+
+* Install [Go](https://go.dev/dl/) (1.20 or higher) from source or use homebrew  `brew install go`
```
```suggestion
* Install [Go](https://go.dev/dl/) (1.20 or higher) from source or use Homebrew  
```shell
brew install go
```
```

o11ika commented on src/install/aws.md:
```diff
@@ -0,0 +1,141 @@
+
+### System Requirements
+
+Make sure that your developer environment has the following. This page assumes that you’re using bash. Adapt configuration and commands as necessary for your preferred shell.
```
> The default shell used in macOS Terminal has varied over different versions of macOS. For a long time, macOS used Bash (Bourne Again SHell) as the default shell. However, starting from macOS Catalina (version 10.15), released in October 2019, macOS switched the default shell from Bash to Zsh (Z Shell).

Should we just say that this is Mac setup?

o11yguru committed:
task: updating ingress

weird-bytecode committed:
add toc, validate data flow and console screenshots

weird-bytecode committed:
little formatting cleanups, 🐙🎉

o11yguru committed:
task: update more docs

weird-bytecode committed:
Merge branch 'documentation-updates' into wb/validate-data-flow

weird-bytecode committed:
Merge pull request #4 from DecisiveAI/wb/validate-data-flow

add toc, validate data flow and console screenshots

o11ika committed:
update AWS install

o11yguru committed:
task: updating doc sections

o11ika committed:
tiny update

o11ika committed:
rename

o11ika committed:
one more

o11yguru committed:
task: updating doc sections

o11yguru committed:
task: updates for joe

o11ika committed:
update otel pre configuration

weird-bytecode committed:
add console page

o11ika committed:
add google analytics

o11yguru committed:
task: updating docs in a more chapter-friendly way

o11yguru committed:
fix: post-demo doc updates

zoomzoomsheim committed:
Update SUMMARY.md

zoomzoomsheim committed:
Update mdai-console.md

Work in progress. figuring out styles

zoomzoomsheim committed:
Update mdai-console.md

zoomzoomsheim committed:
Update mdai-console.md

Copy changes made. 
NOTE: needs the second Sankey Diagram screenshot replaced with highlighted version (noted in CSS)

o11ika committed:
update for the local installation script

o11ika committed:
add context switch step for local unistall

o11ika committed:
make it a note

checosodecisive committed:
Update start.md

checosodecisive committed:
Update start.md

checosodecisive committed:
Update prerequisites.md

checosodecisive committed:
Update prerequisites.md

checosodecisive committed:
Update aws-sso.md

checosodecisive committed:
Update configure.md

tsloughter commented on src/intro/architecture.md:
```diff
@@ -0,0 +1,44 @@
+
+Here's a brief diagram for how we are using our preferred technologies to build our infrastructure to help the DevOps community!
+
+[![Container Diagram](../media/arch-c2.png)](../media/arch-c2.png)
```
I think a paragraph describing what the API is doing and what the Console does below here would be useful. 

I'd also say the MyDecisiveAI octopus outside of the k8s cluster with an arrow saying "monitors" and "configures" is confusing. It could give the impression that MyDecisive is a cloud services running elsewhere monitoring your deployment.

tsloughter commented on src/install/aws/destroy.md:
````diff
@@ -0,0 +1,42 @@
+  ```shell
+  cdk destroy --profile <your_aws_profile>
+  ```
+- The destroy process will run for a while and may return an error due some resources having dependencies.
````
```suggestion
- The destroy process will run for a while and may return an error due to some resources having dependencies.
```

tsloughter commented on src/install/aws/destroy.md:
````diff
@@ -0,0 +1,42 @@
+
+- Delete the Open Telemetry Collector
+  ```shell
+  kubectl delete otelcol/<your_collector_name>
````
Where is this name from?

checosodecisive committed:
Update installation.md

checosodecisive committed:
Update installation.md

checosodecisive committed:
Update start.md

checosodecisive committed:
Update SUMMARY.md

checosodecisive committed:
Update deploy.md

checosodecisive committed:
Update deploy.md

checosodecisive committed:
Update ingress.md

checosodecisive committed:
Update verify.md

checosodecisive committed:
Update SUMMARY.md

checosodecisive committed:
Update verify.md

checosodecisive committed:
Update generate-telemetry.md

tsloughter commented on src/install/aws/configure.md:
````diff
@@ -0,0 +1,105 @@
+```
+cd mdai-inkops
+
+code mdai-inkops
````
I'd remove any reference to a specific IDE. I assume this is VSCode?
